hr_attendance_rest_time_included/models/hr_attendance_reason.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2025 Tecnativa - Eduardo Ezerouali
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import fields, models
+
+
+class HrAttendanceReason(models.Model):
+    _inherit = "hr.attendance.reason"
+
+    include_rest_time = fields.Boolean(default=False)
```

Comment:
And the default is False already, so no need to put it.


```suggestion
    rest_time_included = fields.Boolean()
```